source/docs/hardware-components/fasteners-guide.rst

```diff
@@ -0,0 +1,182 @@
+..    include:: <isonum.txt>
+
+Fasteners Guide
+===============
+If you want your robot to stay in one piece, you will need fasteners. Fasteners are any pieces of hardware that mechanically join parts. This guide will discuss common types of fasteners, and how to use them on your robot.
+
+Threaded Fasteners
+------------------
+Bolts, screws and nuts are all threaded fasteners -- meaning they use a helical thread to join parts.
+
+Thread Basics
+^^^^^^^^^^^^^
+To understand threaded fasteners, you will need to know some basic terminology
+
+
+.. figure:: images/thread-diagram.png
+   :alt: diagram showing thread terminology
+
+External thread
+    A thread on an outside surface. Bolts have external threads
+Internal thread
+    A thread on an inside surface. Nuts have internal threads
+Thread pitch
+    The distance from one thread to another, measured along the length of the thread. This can also be expressed as the number of threads per unit of length, for example threads per inch
+Major diameter
+    The larger of the two diameters giving the height of the thread profile.
+Minor diameter
+    The smaller of the two diameters giving the height of the thread profile.
```

Comment:
```suggestion
External Thread
    A thread on an outside surface. For example, bolts have external threads.
Internal Thread
    A thread on an inside surface. For example, nuts have internal threads.
Thread Pitch
    The distance from one thread to another, measured along the length of the thread. This can also be expressed as the number of threads per unit of length, for example threads per inch.
Major Diameter
    The largest diameter of the thread profile.
Minor Diameter
    The smallest diameter of the thread profile.
```